diagnostic_rt_qpcr/libraries/diagnosticrtqpcrcompositions/source.rb

```diff
@@ -3,10 +3,17 @@
 
 needs 'PCR Libs/PCRComposition'
 
+# Not sure I like that this knows of PCRCompositionDefinitions
```

Comment:
It's possible to pass in the hash when creating the PCRComposition

Reply:
It would probably be better to move the COVID-specific definitions to a new file in this category.

Reply:
Doesn't actually need to access PCRComposition anymore.  I corrected that and forgot to remove this